tests/transformer/aws_test.py

```diff
@@ -18,6 +18,13 @@ def test_aws_transformer_command(runner, tmp_path):
         ],
     )
 
+    if "0" != f"{result.exit_code}":
```

Comment:
You should be able to do just `assert result.exit_code == 0`. Pytest should print the exit code (and possibly the result) for you if the check fails. Ideally, you could write a helper function that would run the transformer and do some default checks of the result. You can see in this commit that there is a pattern that you repeat in each test. That's problematic when you want to do some systematic changes in the tests (like this one).

Reply:
Maybe I don't fully got it, but this thing is there to print `result.output` in a case of an error. This output helped me to understand the reason for this non zero error code. Does this make sense? :)

Reply:
That totally makes sense, but it can be simplified :-) For example, you can achieve the same thing with the `assert condition, message` statement:

```python
assert result.exit_code == 0, f"expected no error, but got code {result.exit_code} and output:\n{result.output}"
```

Why it is better?
* Don't convert values to other types if it is not necessary. If you expect an integer, let's check an integer.
* The body of the `if` statement is not run when the test succeeds. However, the test should usually succeed. That means that this code doesn't have to work and you won't know that until the test breaks. Basically, you have no code coverage.
* It is easier to read.

Reply:
Ah nice! Thanks a lot - I will address this :smile: I didn't know it was possible that way :+1:

Reply:
Follow up PR: https://github.com/redhatcloudx/transformer/pull/612